raiden-dapp/tests/e2e/specs/scenario.spec.ts

```diff
@@ -1,73 +1,18 @@
-import {
-  navigateToDisclaimer,
-  navigateToSelectHub,
-  navigateToNotificationPanel,
-  navigateToAccountMenu,
-  navigateToBackupState,
-  navigateToTokenSelect,
-  navigateToConnectNewToken,
-  navigateBackToTransferScreen,
-  navigateToTokenDeposit,
-  navigateToChannelsList,
-  navigateToRaidenAccount,
-  navigateToWithdrawal,
-  navigateToUDC,
-  navigateToDownloadLogs
-} from '../utils/navigation-utils';
-import {
-  acceptDisclaimer,
-  connectToDApp,
-  enterAndSelectHub,
-  enterChannelDepositAmount,
-  deleteTopNotification,
-  closeNotificationPanel,
-  enterTransferAddress,
-  enterTransferAmount,
-  makeDirectTransfer,
-  downloadState,
-  depositTokensToOpenedChannel,
-  withdrawTokens,
-  withdrawUDCTokens
-} from '../utils/user-interaction-utils';
-import {
-  mintAndDepositUtilityToken,
-  mintSelectedToken,
-  openChannel,
-  transferETHToRaidenAccount,
-  withdrawTokensBackToMainAccount
-} from '../utils/blockchain-interaction-utils';
+import { navigateToDisclaimer } from '../utils/navigation';
+import { acceptDisclaimer, connectToDapp } from '../utils/user-interaction';
 
-it('', () => {
-  navigateToDisclaimer();
-  acceptDisclaimer();
-  connectToDApp();
-  navigateToSelectHub();
-  mintAndDepositUtilityToken();
-  mintSelectedToken();
-  enterAndSelectHub();
-  enterChannelDepositAmount();
-  openChannel();
-  navigateToNotificationPanel();
-  deleteTopNotification();
-  closeNotificationPanel();
-  enterTransferAddress();
-  enterTransferAmount();
-  makeDirectTransfer();
-  navigateToAccountMenu();
-  navigateToBackupState();
-  downloadState();
-  navigateToTokenSelect();
-  navigateToConnectNewToken();
-  navigateBackToTransferScreen();
-  navigateToTokenDeposit();
-  depositTokensToOpenedChannel();
-  navigateToChannelsList()
-  withdrawTokens();
-  navigateToRaidenAccount();
-  transferETHToRaidenAccount();
-  navigateToWithdrawal();
-  withdrawTokensBackToMainAccount();
-  navigateToUDC();
-  withdrawUDCTokens();
-  navigateToDownloadLogs();
+describe('dApp e2e tests', () => {
+  before(async () => {
+    const allIndexedDBs = await window.indexedDB.databases();
+    allIndexedDBs.forEach((indexedDB) => {
+      window.indexedDB.deleteDatabase(indexedDB.name);
+    });
+  });
```

Comment:
I thought it is recommended to do such things in `support/index.ts`? :thinking: 
There is works for every test suite automatically. Does that not work? Or do you prefer this solution because of other reasons? :smiley:

Reply:
I did not try adding it in the `support/index.ts`. I read in the docs and went with the above solution. Also, it is kinda nice to clear the databases in the before block because then I only have to open the test file to understand what is happening instead of poking around in the support dir. :)